testing/integration/test_base.py

```diff
@@ -0,0 +1,67 @@
+"""Integration test base class for Forch"""
```

Comment:
I don't think "test_base" is the best for this since it's ambiguous with the other test_base we already have.  How about "base_integration"? Or "integration_base"?

Reply:
Done